tests/sqlparser_common.rs

```diff
@@ -1559,65 +1564,6 @@ fn parse_select_group_by() {
     );
 }
 
-#[test]
-fn parse_select_group_by_grouping_sets() {
```

Comment:
these tests are postgres specific (test with generic dialect and postgres) so moved to sqlparser_postgres